src/Components/TodoList/ListItem/ListItem.tsx

```diff
@@ -1,9 +1,12 @@
-import React from 'react'
+import { Box, Button, Checkbox, Typography } from "@mui/material";
+import React from "react";
 
 export default function ListItem() {
   return (
-    <div>
-      list item
-    </div>
-  )
+    <Box sx={{ display: "flex", flex: 1, alignItems: "center" }}>
+      <Checkbox />
+      <Typography sx={{ display: "flex" , width: '100vh'}}>option</Typography>
```

Comment:
A few things here:
1. We want to develop generic components, ones that can be used on multiple occasions.
2. The text "option" should not be a magic value, it should be passed via a prop.
3. the width value should not have the "vh" unit, "vh" stands for viewport height - 1% of the height of the display, this is width we are talking about so "vw (viewport width)" would be appropriate. in any case, you should not provide a literal value here, this should be dependent on the width of the parent component, "100%" would be best here. make sure the parent have a height property so that this value will be applied.